src/doxygen.cpp

```diff
@@ -8379,7 +8379,7 @@ static void findDefineDocumentation(Entry *root)
       else
       {
         warn(root->fileName,root->startLine,
-             "found documented #define but ignoring it because "
+             "found documented #define %s but ignoring it because "
              "ENABLE_PREPROCESSING is NO.\n",
```

Comment:
Looks like the meaning changed here.

Reply:
I assumed that `root->name.data` contains the name of the define. To be honest, I didn't verify that, but assumed it wouldn't make things worse, as file and line are printed, but might help if I was correct. ;)

Reply:
After rechecking, this looks like to be better, warning changed but is better. Looks like due to the minimal view github gives I missed the `root->name.data()` and seen what we see above this is much better.